File: generator/generator.pl
#!/usr/bin/env perl -w

use strict;
{
  my $xsl = "../dml2html.xsl";
  my $input = "../temp/example.xml";
  my $output = "../temp/example.html";
  my $params = "";
  # $params .= "debug=false";
  # $params .= " toc.skipped.sections=0";
  # $params .= " status.hidden.values='xxx yyy zzz'";
  `java net.sf.saxon.Transform -dtd:off -versionmsg:off -s:$input -xsl:$xsl -o:$output $params`;
}

File: generator/generator.rb
#!/usr/bin/env ruby

LOGFILE = "generator.log"
@xsl = "../dml2html.xsl"

@input = "../temp/example.xml"
@output = "../temp/example.html"

begin
  system "java -jar /Library/Java/Extensions/saxon91005/saxon9.jar -dtd:off -versionmsg:off -s:#{@input} -xsl:#{@xsl} -o:#{@output} 2>> #{LOGFILE}"
  
rescue => err
  puts "Exception: #{err}"
  err
end